backend/src/main/java/com/bakdata/conquery/io/jackson/Jackson.java

```diff
@@ -41,23 +42,25 @@
 			.enable(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES)
 			.enable(DeserializationFeature.FAIL_ON_UNRESOLVED_OBJECT_IDS)
 			.enable(DeserializationFeature.USE_BIG_DECIMAL_FOR_FLOATS)
-			.disable(SerializationFeature.WRITE_EMPTY_JSON_ARRAYS)
 			.disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS)
 			.setLocale(Locale.ROOT)
 			.disable(JsonGenerator.Feature.AUTO_CLOSE_TARGET)
+			.enable(SerializationFeature.WRITE_EMPTY_JSON_ARRAYS)
```

Comment:
Why do we use the deprecated feature and do not control it over the serialization inclusion?

Reply:
Weil das viel aufwändiger wäre das an jeder Stelle zu machen anstatt es per default umzustzen. Es ist schlichtweg auch das bessere Default für gesundes API Parsing, sonst muss @Kadrian an jeder Stelle testen wie wo was für Daten stehen.